PyInstaller/exceptions.py

```diff
@@ -12,3 +12,30 @@
 
 class ExecCommandFailed(SystemExit):
     pass
+
+
+class HookError(Exception):
+    """base class for hook related errors"""
+    pass
+
+
+class ImportErrorWhenRunningHook(HookError):
+    def __str__(self):
+        return ("Failed to import module {} required by hook for module {}. "
+                "Please check whether module {} actually exists and whether "
+                "the hook is compatible with your version of {}: You might "
+                "want to read more about hooks in the manual and provide "
+                "a pull-request to improve PyInstaller.".format(
+                    self.args[0], self.args[1], self.args[0], self.args[1]))
```

Comment:
Hi, with your indentation removed flask8 returns the following: 

`(pyinstaller) riz@riz-desktop:~/Desktop/projects/pyinstaller$ flake8 /home/riz/Desktop/projects/pyinstaller/PyInstaller/exceptions.py 
/home/riz/Desktop/projects/pyinstaller/PyInstaller/exceptions.py:29:17: E122 continuation line missing indentation or outdented
                self.args[0], self.args[1]))
`

Would you like me to ignore flask8 and remove the indentation as per your suggestion?

Reply:
@RS357 Use the `{0}`s and `{1}`s as @Legorooj suggests but indent the  `self.args...` to get flake8 to accept it. Github doesn't lint check proposed changes - otherwise @Legorooj would've probably indented it himself.

Reply:
@RS357 Can you mark this as resolved as it is resolved.